.github/ISSUE_TEMPLATE/upgrade_regression_form.yml

```diff
@@ -0,0 +1,133 @@
+name: ⬆️ Upgrade - Build Regression
+description: If you are upgrading to a new React Native version (stable or pre-release) and encounter a build regression.
+labels: ["Needs: Triage :mag:", "Type: Upgrade Issue"]
```

Comment:
I think we can probably remove the second label

Reply:
Yup the second label can go 👍